time stamp
uint32 states
float32[] phases
float32[] phaseVelocities
float32[] activations

